wiki/Client/Release_stream/Lazer/File_storage/en.md

```diff
@@ -0,0 +1,48 @@
+# File storage in osu!(lazer)
+
+By default, osu!(lazer) stores beatmaps, skins, and score replay files in the following directories:
+
+- `%appdata%/osu/files` (Windows),
```

Comment:
just noticed that this is incorrect, it's `%appdata%/roaming/osu/files` iirc

Reply:
`%appdata%` already leads to `C:\Users\$USERNAME\AppData\Roaming`. As opposed to `%localappdata%` which leads to `C:\Users\$USERNAME\AppData\Local`. See https://learn.microsoft.com/en-us/windows/deployment/usmt/usmt-recognized-environment-variables (`CSIDL_APPDATA` vs `CSIDL_LOCAL_APPDATA`).